packages/edit-site/src/components/block-editor/editor-canvas.js

```diff
@@ -31,6 +39,30 @@ function EditorCanvas( { enableResizing, settings, children, ...props } ) {
 	const { setCanvasMode } = unlock( useDispatch( editSiteStore ) );
 	const deviceStyles = useResizeCanvas( deviceType );
 	const mouseMoveTypingRef = useMouseMoveTypingReset();
+	const [ isFocused, setIsFocused ] = useState( false );
+
+	useEffect( () => {
+		if ( canvasMode === 'edit' ) {
+			setIsFocused( false );
+		}
+	}, [ canvasMode ] );
```

Comment:
Hi, @jeryj

Sorry, odd question: I came across this code and wondered how to reproduce the state when this effect is needed.

Reply:
It's a good question! I wasn't sure off the top of my head either 😅  I took a look, and without it, the `is-focused` classname isn't removed when pressing enter on the editor frame to switch to edit mode. When you return back to the sidebar view, the `is-focused` class is still there.

To see this behavior, remove this code, then:

- Start at the site editor with sidebar open
- Using the tab key to move focus to the editor frame. The `is-focused` class is what adds the blue focus ring on the iframe since that element doesn't want to deal with focus events or `:focus` selector 😒 
- Press enter to switch to edit mode
- Press the W logo to return to the sidebar view
- The focus is on the W logo, and the `is-focus` class is incorrectly still on the editor iframe

I'm attempting to upload a screen recording with it, but I'm at a coffee shop and it's not working...
Uploading Screen Recording 2023-07-13 at 2.02.52 PM.mov…

Reply:
Thank you, @jeryj 🙇